src/libaktualizr/primary/sotauptaneclient.h

```diff
@@ -55,6 +56,7 @@ class SotaUptaneClient {
   std::shared_ptr<PackageManagerInterface> pacman;
   HttpInterface &http;
   Uptane::Fetcher uptane_fetcher;
+  const Bootloader &bootloader;
```

Comment:
Is there a reason not to make this a `shared_ptr`?

Reply:
Not really. I think until now we only wrapped stuff into `shared_ptr` when we really needed to. But I agree that we might want to make it standard. Should we pass `HttpInterface` and `Uptane::Repository` as shared pointers as well or were there some reasons not to do it?

Reply:
Good point.  I'm in favour of std::shared_ptr over references everywhere, but for consistency with the other parameters we can/should stick with references here.  The move away from references can be a refactor :)